srcalc/src/main/java/gov/va/med/srcalc/domain/model/DiscreteNumericalVariable.java

```diff
@@ -222,6 +222,36 @@ final void setUpperBound(final float upperBound)
         {
             fUpperBound = upperBound;
         }
+        
+        /**
+         * Returns the upper bound as a String.
+         */
+        @Basic
+        final String getUpperBoundString()
+        {
+            return Float.toString(fUpperBound);
+        }
+        
+        /**
+         * Sets the upper bound for this category.
+         * @param upperBoundString the String to parse into the upper bound
+         * @throws NumberFormatException if {@code upperBoundString} cannot be parsed into a float
+         */
+        final void setUpperBoundString(final String upperBoundString)
+        {
+            try
+            {
+                fUpperBound = Float.parseFloat(upperBoundString);
+            }
+            catch(final NumberFormatException e)
+            {
+                // Log the exception but still throw an exception because the method contract
+                // should not allow invalid values.
+                LOGGER.debug("Unable to parse the String into a float for an "
```

Comment:
Please ensure the attempted value is included somewhere in the log (either via this statement or the ultimate exception handler). This will be essential for locating bad values in the database.